client/components/App.jsx

```diff
@@ -69,23 +73,24 @@ class App extends Component {
             <ProductDetail
               product={currentProduct}
               updateProduct={(id) => this.updateProduct(id)}
-              productReviews={this.state.reviewData}
+              productReviews={reviewData}
+              addToCart={(obj) => this.addToCart(obj)}
             />
-            {this.state.relatedProducts &&
+            {this.state.relatedProducts && (
               <RelatedItems
                 product={currentProduct}
                 related={this.state.relatedProducts}
                 updateProduct={(id) => this.updateProduct(id)}
                 rating={this.state.reviewData}
               />
-            }
+            )}
             <QuestionsAnswers
               product={currentProduct}
               updateProduct={(id) => this.updateProduct(id)}
             />
-            {this.state.reviewData && (
+            {reviewData && (
```

Comment:
what are the values of these variables? careful not to land on a _falsy_ value (i.e. `reviewData` is a number that can be `0`), otherwise you won't get what you're looking for